### DISCUSSION TITLE
adding RoomPlan API

### DISCUSSION THREAD

**Discussion by macrozone** [ViroCommunity/viro#169] [Ideas]
Anyone toyed around with https://developer.apple.com/augmented-reality/roomplan/ ? would that be feasible to add to viro AR (at least on ios)?

Background: 
I maintained the old react-native-arkit (https://github.com/react-native-ar/react-native-arkit) 5 years ago, but because there was no project anymore where i could use it, i had to drop it. Viro looks still maintained and I finally got another potential AR case, so it might be a good match. The case would require roomplan API though (at least initially). Maybe I could even contribute a bit, but i would first like to now if its even feasible to add roomplan API

**Comment by harmit001** [comment]
@macrozone Did you find any alternative then please let me know.

**Reply by macrozone** [reply]
hey, did not yet check yet, just wanted to gather some input here.
